tools/star.wdl

```diff
@@ -540,7 +540,7 @@ task alignment {
         Float outFilterMatchNminOverLread = 0.66
         Float scoreGenomicLengthLog2scale = -0.25
         Float seedSearchStartLmaxOverLread = 1.0
-        Float alignSplicedMateMapLminOverLmate = 0.66
+        Float alignSplicedMateMapLminOverLmate = 0.5
```

Comment:
This has very little information in the manual, so I'm not sure what it's doing. I see the value changes very little, but do you know if Arriba introduces any reasoning as to why they changed this?

Reply:
Okay, I see. Since we are balancing all of these concerns based on the purpose, see my general comment below on whether the defaults can match STAR defaults for this task but then we can override them for Arriba-based workflows.